src/auth/aws.test.js

```diff
@@ -4,24 +4,32 @@ import sinon from 'sinon';
 import * as utils from './utils';
 import * as consts from '../consts';
 import * as _ from './_aws';
-import {
-    createCanonicalRequest,
-    // createStringToSign,
-} from './aws';
+import { createCanonicalRequest, createStringToSign, createSigningKey, createSignature, createAuthorizationHeader } from './aws';
+
 
 /**
  * 'PRIVATE' methods
  */
 
 test('getAWSTimestamp returns right format', (t) => {
-    const clock = sinon.useFakeTimers(new Date('2017-07-11T09:46:56Z'));
+    const clock = sinon.useFakeTimers(new Date('2017-07-11T09:46:56Z').valueOf());
```

Comment:
Do not like. My version was from the docs.

why are you getting different things turning up to what I'm getting?